platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java

```diff
@@ -1253,7 +1253,7 @@ public void componentResized(ComponentEvent event) {
         if (height != myHeight) {
           myHeight = height;
           myResizeInProgress = true;
-          myErrorText.setMinimumSize(new Dimension(0, height));
+          myErrorText.setMinimumHeight(height);
```

Comment:
This change is redundant. It technically changes nothing.

Reply:
Removeed